linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_files_permissions/tests/perms_0600.pass.sh

```diff
@@ -22,4 +25,12 @@ cat << EOF > $RSYSLOG_CONF
 
 *.*        ${RSYSLOG_TEST_LOGS[0]}
 
+ *.*        ${RSYSLOG_TEST_LOGS[1]}
```

Comment:
I think that there is a problem that the OVAL for this rule always returns false if `/etc/rsyslog.conf` doesn't contain any inlcusion. Please add `\$IncludeConfig /etc/rsyslog.d/*.conf` to both test scenarios.

Reply:
@matejak this is the problem

Reply:
> I think that there is a problem that the OVAL for this rule always returns false if `/etc/rsyslog.conf` doesn't contain any inlcusion.

This may be an actual finding in the OVAL check.
If `rsyslog.conf` or `/etc/rsyslog.d/*.conf` without includes are correct, the OVAL check should just pass.

@jvymazal Are includes for `rsyslog` `*.conf` file mandatory?

Reply:
After consulting with @jvymazal, `rsystlog` will run even if there is no include directive present.
As an interesting fact, in a default installation, the directive is there, but it doesn't do anything, because the `/etc/rsyslog.d/` directory is empty.